### PR TITLE
[1LP][RFR] Configuring remote app. and global App replication using UI

### PR DESCRIPTION
**Pull request by dgaikwad**
## Purpose or Intent

- __Adding tests__ Adding new tc which is making remote appliance, global appliance and starting replication using UI
## Some examples are:
- {{pytest: cfme/tests/test_replication.py::test_replication_appliance_set_type_global_ui --long-running -v}}

### CODE REVIEW

File: cfme/tests/test_replication.py [hunk 1]
```diff
@@ -198,7 +198,38 @@ def test_replication_appliance_set_type_global_ui():
...
+    navigate_to(remote_region.replication, "RemoteAdd")
+    remote_region.replication.set_replication(replication_type="remote")
+
+    # Adding Remote Appliance into Global appliance using UI
+    global_region = global_app.collections.regions.instantiate(number=99)
+    navigate_to(global_region.replication, "Global")
+    global_region.replication.set_replication(replication_type="global",
+                                       updates={"host": remote_app.hostname},
+                                       validate=True)
+
```
Comment (ganeshhubale) [inline cfme/tests/test_replication.py]: Improve formatting:
```
global_region.replication.set_replication(
        replication_type="global", updates={"host": remote_app.hostname}, validate=True
    )
```

File: cfme/tests/test_replication.py [hunk 2]
```diff
@@ -198,7 +198,38 @@ def test_replication_appliance_set_type_global_ui():
...
+    # Validating replication
+    assert global_region.replication.get_replication_status(host=remote_app.hostname),\
+        "Replication is not started."
```
Comment (ganeshhubale) [inline cfme/tests/test_replication.py]: same here

File: cfme/configure/configuration/region_settings.py [hunk 3]
```diff
@@ -1038,21 +1038,31 @@ def set_replication(self, updates=None, replication_type=None, reset=False, vali
...
-                view.save_button.click()
+                if replication_type == 'global':
+                    if validate:
+                        view.action_dropdown.item_select("Validate")
+                        view.flash.assert_success_message(
+                            "Subscription Credentials validated successfully"
+                        )
+                    view.accept_button.click()
+                    view.save_button.click()
+                    view.flash.assert_success_message('Replication configuration save initiated.'
```
Comment (valaparthvi) [inline cfme/configure/configuration/region_settings.py]: We do not want the flash messages here in view. We can do that assertion in the test cases if required.

File: cfme/configure/configuration/region_settings.py [hunk 4]
```diff
@@ -1038,21 +1038,31 @@ def set_replication(self, updates=None, replication_type=None, reset=False, vali
...
+                                              ' Check status of task "Save subscriptions for '
+                                              'global region" on My Tasks screen')
+                elif replication_type == 'remote':
+                    view.save_button.click()
+                    view.flash.assert_success_message('Replication configuration save initiated.'
+                                                      ' Check status of task "Configure the '
+                                                      'database to be a replication remote region"'
+                                                      ' on My Tasks screen')
```
Comment (valaparthvi) [inline cfme/configure/configuration/region_settings.py]: Same as above.

File: cfme/tests/test_replication.py [hunk 5]
```diff
@@ -198,7 +198,38 @@ def test_replication_appliance_set_type_global_ui():
...
+        sshlogin=credentials["ssh"]["username"],
+        sshpass=credentials["ssh"]["password"],
+    )
+
+    global_app.appliance_console_cli.configure_appliance_internal_fetch_key(**app_params)
+    global_app.evmserverd.wait_for_running()
+    global_app.wait_for_web_ui()
+
+    # Making configured app to Remote Appliance using UI
+    remote_region = remote_app.collections.regions.instantiate()
```
Comment (valaparthvi) [inline cfme/tests/test_replication.py]: Will you need to specify the region number here?

Comment (dgaikwad) [reply cfme/tests/test_replication.py]: The default value is 0 for region and I am also expecting the same so no need to provide value like below:
 `remote_region = remote_app.collections.regions.instantiate(number=0)`

File: cfme/tests/test_replication.py [hunk 6]
```diff
@@ -198,7 +198,37 @@ def test_replication_appliance_set_type_global_ui():
...
+    remote_region.replication.set_replication(replication_type="remote")
+
+    # Adding Remote Appliance into Global appliance using UI
+    global_region = global_app.collections.regions.instantiate(number=99)
+    navigate_to(global_region.replication, "Global")
+    global_region.replication.set_replication(
+        replication_type="global", updates={"host": remote_app.hostname}, validate=True)
+
+    # Validating replication
+    assert global_region.replication.get_replication_status(
```
Comment (valaparthvi) [inline cfme/tests/test_replication.py]: **Optional:** Improve formatting here
```suggestion
    assert global_region.replication.get_replication_status(
        host=remote_app.hostname
    ), "Replication is not started."
```

File: cfme/tests/test_replication.py [hunk 7]
```diff
@@ -198,7 +198,37 @@ def test_replication_appliance_set_type_global_ui():
...
+        sshpass=credentials["ssh"]["password"],
+    )
+
+    global_app.appliance_console_cli.configure_appliance_internal_fetch_key(**app_params)
+    global_app.evmserverd.wait_for_running()
+    global_app.wait_for_web_ui()
+
+    # Making configured app to Remote Appliance using UI
+    remote_region = remote_app.collections.regions.instantiate()
+    navigate_to(remote_region.replication, "RemoteAdd")
```
Comment (digitronik) [inline cfme/tests/test_replication.py]: No need of this navigation. `set_replication` will do it for you
https://github.com/ManageIQ/integration_tests/pull/9738/files#diff-7bd4eae0f9d6d82836427a85dacdb811R1042

File: cfme/tests/test_replication.py [hunk 8]
```diff
@@ -198,7 +198,37 @@ def test_replication_appliance_set_type_global_ui():
...
+    remote_region.replication.set_replication(replication_type="remote")
+
+    # Adding Remote Appliance into Global appliance using UI
+    global_region = global_app.collections.regions.instantiate(number=99)
+    navigate_to(global_region.replication, "Global")
```
Comment (digitronik) [inline cfme/tests/test_replication.py]: same ^^ why we need this?

**Pull request merged**